### PR TITLE
PHOENIX-5063 Create a new repo for the phoenix query server

### PR DESCRIPTION
**Pull request by karanmehta93**

### CODE REVIEW

File: queryserver-client/pom.xml [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.apache.phoenix</groupId>
+    <artifactId>phoenix-queryserver</artifactId>
+    <version>4.15.0-HBase-1.4-SNAPSHOT</version>
```
Comment (twdsilva) [inline queryserver-client/pom.xml]: change this to 1.0.0-SNAPSHOT

File: queryserver/pom.xml [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.apache.phoenix</groupId>
+    <artifactId>phoenix-queryserver</artifactId>
+    <version>4.15.0-HBase-1.4-SNAPSHOT</version>
```
Comment (twdsilva) [inline queryserver/pom.xml]: change this to 1.0.0-SNAPSHOT

**Comment by karanmehta93** [comment]
@twdsilva @joshelser 
Sorry for getting back late on this one. 
The current PR will now create a tar ball that contains all the required jars and scripts and can launch queryserver independently. Please have a look.

**Comment by twdsilva** [comment]
+1 LGTM, nice work.

**Pull request merged**